globus_automate_client/client_helpers.py

```diff
@@ -0,0 +1,82 @@
+from typing import Optional
+
+from globus_automate_client.action_client import ActionClient
+from globus_automate_client.flows_client import (
+    MANAGE_FLOWS_SCOPE,
+    PROD_FLOWS_BASE_URL,
+    FlowsClient,
+)
+from globus_automate_client.token_management import get_cli_authorizer
```

Comment:
Seems good having `cli` in the name to make this explicit.

Reply:
I was sorely tempted to move the whole of `globus_automate_client.token_management` into the `cli` directory but opted not to to keep diffs manageable and to reduce the potential for breakage.